### PR TITLE
Updates to improve the what is kubernetes page.

### LINKED ISSUE #12333: Improve "What is Kubernetes?" page
**This is a...** 

- [x] Feature Request
- [ ] Bug Report

**Problem:**
The current "What is Kubernetes?" page could be improved to better convey what Kubernetes is, where it fits into the Cloud as-a-service continuum, and lead into the conceptual architectural overview.

**Proposed Solution:**
Rewrite the page using conceptual models established by the Fundamental Modeling Concepts project. 

**Page to Update:**
https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/

### PR DESCRIPTION
**Pull request by bradtopol**
This PR focuses on improving the what is Kubernetes section of the documentation.
fixes #12333

Signed-off-by: Brad Topol <btopol@us.ibm.com>

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by bgrant0607** [comment]
/hold

Thanks, but I have some issues with these changes, and don't have time for a detailed review right now.

File: content/en/docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -15,16 +15,26 @@ This page is an overview of Kubernetes.
 {{% /capture %}}
 
 {{% capture body %}}
-Kubernetes is a portable, extensible open-source platform for managing
-containerized workloads and services, that facilitates both
-declarative configuration and automation. It has a large, rapidly
-growing ecosystem. Kubernetes services, support, and tools are widely available.
+Kubernetes is a cloud infrastructure that provides for the deployment and
```
Comment (sftim) [inline content/en/docs/concepts/overview/what-is-kubernetes.md]: Is Kubernetes an infrastructure?

I think of an infrastructure (in the IT sense) as something that I can run a specific workload on. I _can_ run a workload on a Kubernetes infrastructure, and that might well be in the cloud.

File: content/en/docs/concepts/overview/what-is-kubernetes.md [hunk 2]
```diff
@@ -15,16 +15,26 @@ This page is an overview of Kubernetes.
...
+orchestration of containerized applications.    
+It is a portable and extensible open-source platform that is capable of 
+managing both containerized workloads and services. Kubernetes
+provides key features such as a declarative configuration and deployment model,
+built in support for application replicas and autoscaling of applications, and
+also provides support for deploying multiple versions of an application.  
+In addition, Kubernetes has built in health-checking support for its applications,
```
Comment (sftim) [inline content/en/docs/concepts/overview/what-is-kubernetes.md]: This sentence starts “In addition” and ends “as well”. I think just one of those would be enough to make the point.

File: content/en/docs/concepts/overview/what-is-kubernetes.md [hunk 3]
```diff
@@ -15,16 +15,26 @@ This page is an overview of Kubernetes.
...
+and networking and volume support as well. 
 
 Google open-sourced the Kubernetes project in 2014. Kubernetes builds upon
 a [decade and a half of experience that Google has with running
 production workloads at
 scale](https://research.google.com/pubs/pub43438.html), combined with
-best-of-breed ideas and practices from the community.
+best-of-breed ideas and practices from the community.  Kubernetes has a large 
+and rapidly growing ecosystem. Kubernetes is now available as a hosted managed
```
Comment (sftim) [inline content/en/docs/concepts/overview/what-is-kubernetes.md]: How about “All major public cloud providers offer Kubernetes as a hosted managed service, …”?

File: content/en/docs/concepts/overview/what-is-kubernetes.md [hunk 4]
```diff
@@ -15,16 +15,26 @@ This page is an overview of Kubernetes.
...
-Kubernetes is a portable, extensible open-source platform for managing
-containerized workloads and services, that facilitates both
-declarative configuration and automation. It has a large, rapidly
-growing ecosystem. Kubernetes services, support, and tools are widely available.
+Kubernetes is a cloud infrastructure that provides for the deployment and
+orchestration of containerized applications.    
+It is a portable and extensible open-source platform that is capable of 
+managing both containerized workloads and services. Kubernetes
+provides key features such as a declarative configuration and deployment model,
+built in support for application replicas and autoscaling of applications, and
```
Comment (kbhawkey) [inline content/en/docs/concepts/overview/what-is-kubernetes.md]: nit: you could drop `provides` in:
Kubernetes provides key features and instead just list the key features. Same goes for the clause:
`also provides support for ...`

File: content/en/docs/concepts/overview/what-is-kubernetes.md [hunk 5]
```diff
@@ -15,16 +15,26 @@ This page is an overview of Kubernetes.
...
 Google open-sourced the Kubernetes project in 2014. Kubernetes builds upon
 a [decade and a half of experience that Google has with running
 production workloads at
 scale](https://research.google.com/pubs/pub43438.html), combined with
-best-of-breed ideas and practices from the community.
+best-of-breed ideas and practices from the community.  Kubernetes has a large 
+and rapidly growing ecosystem. Kubernetes is now available as a hosted managed 
+service from all major public cloud providers, and there are several distribution 
+options available for running Kubernetes as an on-premises private cloud as well.
+Furthermore, Kubernetes services, support, and tools are widely available.
```
Comment (kbhawkey) [inline content/en/docs/concepts/overview/what-is-kubernetes.md]: nit: does the clause `and there are several distribution options ...` help others understand what Kubernetes is?
nit: possibly reword or drop `Furthermore`
nit: does widely available mean: on any number of cloud platforms, OSes, computer languages?

**Comment by bradtopol** [comment]
Hi @bgrant0607 would you be able to do a detailed review in the near future?

**Comment by bgrant0607** [comment]
@bradtopol What are you trying to convey that the existing text does not adequately convey? The linked issue is vague, and doesn't link to more background.

File: content/en/docs/concepts/overview/what-is-kubernetes.md (hunk 1 above)
Comment (bradtopol) [reply content/en/docs/concepts/overview/what-is-kubernetes.md]: K. I will change infrastructure to platform

File: content/en/docs/concepts/overview/what-is-kubernetes.md (hunk 2 above)
Comment (bradtopol) [reply content/en/docs/concepts/overview/what-is-kubernetes.md]: Will fix in the next commit

File: content/en/docs/concepts/overview/what-is-kubernetes.md (hunk 3 above)
Comment (bradtopol) [reply content/en/docs/concepts/overview/what-is-kubernetes.md]: Will fix

File: content/en/docs/concepts/overview/what-is-kubernetes.md (hunk 4 above)
Comment (bradtopol) [reply content/en/docs/concepts/overview/what-is-kubernetes.md]: I like having a verb there

Comment (bradtopol) [reply content/en/docs/concepts/overview/what-is-kubernetes.md]: Will fix the provides support for

File: content/en/docs/concepts/overview/what-is-kubernetes.md (hunk 5 above)
Comment (bradtopol) [reply content/en/docs/concepts/overview/what-is-kubernetes.md]: Will fix as requested by a previous reviewer

**Comment by bradtopol** [comment]
> @bradtopol What are you trying to convey that the existing text does not adequately convey? The linked issue is vague, and doesn't link to more background.
Hi @bgrant0607 great question.  So originally both myself and @chenopis signed up to update this page.  I just wanted to improve the readability of the content a little.  @chenopis was going to look at adding some modeling of Kube resources content to improve the content as well. Unfortunately, @chenopis had a role change and was not able to commit to doing the modeling based revisions. So I just did some basic readability improvements and limited the scope to that.

File: content/en/docs/concepts/overview/what-is-kubernetes.md (hunk 4 above)
Comment (kbhawkey) [reply content/en/docs/concepts/overview/what-is-kubernetes.md]: Hi, @bradtopol. Looking good. I like verbs too 😄. I think I meant to say something like,
`The Kubernetes key features are declarative configuration ...`. There are a number of `provides` and `support` sprinkled throughout. Also, when I look at the rendered page now, there appears to be an extra line/spaces before `In addition ...`

**Comment by zacharysarah** [comment]
Closing per discussion in Q2 review/Q3 planning.

/close

**Pull request closed without merging**